xmtp_mls/migrations/2024-08-07-213816_create-private-preference-store/up.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE "private_preferences"(
```

Comment:
Should this be `personal_preferences`? or `private_preferences` ?

Reply:
I thought we settled on private

Reply:
From the topic I actually think we call it `user_preferences`

https://github.com/xmtp/xmtp-android/blob/main/library/src/main/java/org/xmtp/android/library/messages/Topic.kt#L33

Reply:
Oh, you're right. Sorry, we changed this name so many times.

In any case, I'm not sure that should be the name for this table. User preferences are going to be handling more than just consent state (for example, HMAC keys). This table is all about consent state.

Reply:
@neekolas in that case would we make a separate table just for the hmac keys?

Reply:
I think so. I don't think there are a million different cases of private preferences. There's probably 3-5, ever. Might as well store them in a nice way vs one-size-fits-all